### PR TITLE
fixup! [Service Offloading] Key/Mouse Input Support

### PR DESCRIPTION
**Pull request by Sunwoo-Nam**
Fix the build break for ubuntu desktop build.
Remove unnecessary white spaces.

### CODE REVIEW

**Review by yg48 — APPROVED** [review]
LGTM

File: third_party/blink/renderer/modules/input_control/BUILD.gn [hunk 1]
```diff
@@ -35,7 +35,7 @@ if ((is_win || (is_android && enable_service_offloading_knox)) && enable_service
       deps = [ "//base" ]
 
       sources = ["input_control.h"]
-      
+
       if(is_win) {
```
Comment (Chikdolman) [inline third_party/blink/renderer/modules/input_control/BUILD.gn:39]: Do we need enable_service_offloading flag?

Comment (Chikdolman) [reply third_party/blink/renderer/modules/input_control/BUILD.gn:39]: Sorry, I missed line 33.

**Review by Chikdolman — APPROVED** [review]
LGTM

**Pull request merged**